keyword_clustering.py

```diff
@@ -57,14 +55,16 @@ def main(args):
             model = ""
         this_output_file_path = "/".join(output_file_path.split("/")[:-1]) + "/" + \
             file_name + "_" + model + ".tsv"
-        with open(os.path.expanduser(this_output_file_path), 'wt') as f:
-            tsv_writer = csv.writer(f, delimiter='\t')
-            for shortened_keywords, keyword_list in shortened_keywords_list[index].items():
-                if len(keyword_list) > 1:
-                    clustered_keywords += len(keyword_list)        
-                    keyword_list.insert(0, len(keyword_list))
-                    keyword_list.insert(0, shortened_keywords)
-                    tsv_writer.writerow(keyword_list)        
+        
+        f = open(os.path.expanduser(this_output_file_path), 'wt')
+        tsv_writer = csv.writer(f, delimiter='\t')
+        for shortened_keywords, keyword_list in shortened_keywords_list[index].items():
+            if len(keyword_list) > 1:
+                clustered_keywords += len(keyword_list)        
+                keyword_list.insert(0, len(keyword_list))
+                keyword_list.insert(0, shortened_keywords)
+                tsv_writer.writerow(keyword_list)  
+        f.close()
```

Comment:
revert to the original version, sorry about the confusion.

Reply:
Done